wire-ios/Wire-iOS/Sources/Helpers/URL+Wire.swift

```diff
@@ -194,6 +194,11 @@ extension URL {
     static var selfUserProfileLink: URL? {
         BackendEnvironment.selfUserProfileLink
     }
+
+    static var wr_e2eiLearnMore: URL {
+        return wr_support.appendingPathComponent("hc/articles/9211300150685-End-to-end-identity")
```

Comment:
question: does this works for all backend configs? I guess it works for Wire.com but no other specific configs C1...